app/config.py

```diff
@@ -25,6 +26,8 @@ class Config(object):
     # needs to refer to notify for utils
     NOTIFY_LOG_PATH = '/home/vcap/logs/app.log'
 
+    DOCUMENT_DOWNLOAD_COOKIE_AUTH_VALID_FOR = datetime.timedelta(hours=24)
```

Comment:
What’s the rationale for 24 hours? We‘re trying to mitigate against other people accessing the URL from another device, so not sure what benefit we get here over a much longer expiry time.

Reply:
We can make it longer if you want - there's no significantly clear justification or need for it to be short, but it's what I originally put forward and wasn't opposed at the time.

Reply:
Removed the custom expiry.